templates/layout.handlebars

```diff
@@ -17,5 +17,15 @@
     </head>
     <body>
         {{{ content }}}
+
+        {{#if isQuestion }}
```

Comment:
It should, but Hapi Handlebars doesn't support multiple layouts inclusion. This isn't so bad at the moment.